index.ts

```diff
@@ -33,33 +33,15 @@ const rhsUrl = process.env.RHS_URL as string;
 const walletKey = process.env.WALLET_KEY as string;
 const OPID_METHOD = 'opid';
 
-core.registerDidMethod(OPID_METHOD, 0b00000011);
-core.registerDidMethodNetwork({
-  method: OPID_METHOD,
-  blockchain: 'optimism',
-  chainId: 11155420,
-  network: 'sepolia',
-  networkFlag: 0b1000_0000 | 0b0000_0010
-});
-core.registerDidMethodNetwork({
-  method: OPID_METHOD,
-  blockchain: 'optimism',
-  chainId: 10,
-  network: 'main',
-  networkFlag: 0b1000_0000 | 0b0000_0001
-});
```

Comment:
Totally! But don't we need our updated package? I was thinking we can already install it using the GitHub repo URL. Let's add it in this same pr so we keep a functional main branch

Reply:
Yas! Now we have opid-sdk as an npm package :)